src/openapi/api.ts

```diff
@@ -44,11 +44,19 @@ declare namespace Components {
          * payload for push notification delivered to phone
          */
         export interface PushNotificationLightningInvoicePaid {
-            type: 1;
+            /**
```

Comment:
is that what updated `dtsgen` generates now?
its a bit screwed up, as `PushNotificationLightningInvoicePaid` should just have `type` 1